content/blog/how-elevate-supports-business-needs.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "How ELevate supports business needs"
+type: blog
+author: 
+ name: "David der Nederlanden"
+ bio: "Linux Cloud Architect at Bizway"
+ image: /users/ddernederlanden.jpg
+date: '2024-05-22'
+images:
+  - /blog-images/2024/2024-04-23-centos6-to-centos7.png
+post:
+    title: "How ELevate supports business needs"
+    image: /blog-images/2024/2024-04-23-centos6-to-centos7.png
+---
+
+# How Elevate supports business needs
+While we've been busy upgrading all our CentOS 7 servers to AlmaLinux 8 and higher, the ELevate Project now has the abillity to upgrade CentOS 6 servers too!
+
+## whoami and a bit about our systems
+
+I'm David from the Netherlands and I try to spread the feeling I experience myself while working within the hosting industry whenever I can, it really is something magical!
+My employer Bizway, a Dutch Managed Service Provider (MSP from now on) gives me the chance to explore all the different aspects that are on the table when providing services to customers.
+
+Things that interest me a lot are automation, mostly anything opensource, networking and infrastructure.
+
+The first Linux I ever used was actually Debian, which I luckily still use a lot too, as it is the base for [Proxmox Virtual Environment](https://www.proxmox.com).
+While for the past 4 years CentOS and AlmaLinux took most of my time, Debian always kept a special place because of one simple reason, Debian provides in-place OS upgrades.
+
+Even though servers can, and one can discuss, maybe even should be configured in state so you can destroy and replace it easily, that is not always the case.
+
+So when AlmaLinux came around and further down the road ELevate popped up offering in-place upgrades from CentOS 7 systems to for example AlmaLinux 8 and higher, it also entered that special place!
+
+## Why there can be (and are!) still CentOS 6 servers alive and kicking
+We always used CentOS as our main operating system for our webhosting platforms, while we were eventually forced to look elsewhere, we still had a lot of CentOS 7 and a couple of CentOS 6 systems around,
+since ELevate came around we started upgrading our whole fleet to AlmaLinux 8 as we started using that for our new deployments too, we're almost done doing so.
```

Comment:
```suggestion
We always used CentOS as our main operating system for our webhosting platforms, while we were eventually forced to look elsewhere, we still had a lot of CentOS 7 and a couple of CentOS 6 systems around. Since ELevate came out, we started upgrading our whole fleet to AlmaLinux 8 (as we are using that for our new deployments too). We're almost done doing so.
```